nss-day-cohort-48/rare-server-polite-pirates
# update post function

#### Changes Made
1. Modified file `posts/request.py` to include `update_post` functionality.
1. Modified file `request_handler.py` to include `update_post` functionality within `do_PUT` function.


#### Steps to Review
1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-update-post
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > check code for above changes
   > in Postman, do a `PUT` request with the url path `http://localhost:8088/posts/2` , then do a `GET` request for all posts
4. View code file.
   > Confirm file modifications are present as indicated above.
